### PR TITLE
Update global text and field styles

### PR DESCRIPTION
**Pull request by LanesGood**
Multiple styling edits for text and field styles. This is an attempt to match the text and colors used throughout the app more closely to the original UI designs.

@sethvincent can you help out with some of the todos below, especially the first and last item?

TO DO:
~~-[ ] Implement working icons in FeatureDetailHeader for the View and Add point screens~~
- [x] Add [react-native-global-props](https://www.npmjs.com/package/react-native-global-props) or similar to avoid defining text styles on every component and screen
- [x] Test and debug on IOS

### CODE REVIEW

File: app/style/variables.js [hunk 1]
```diff
@@ -18,3 +24,9 @@ export const notificationColors = {
     color: 'black'
   }
 }
+
+export const StyledText = styled.Text`
```
Comment (sethvincent) [inline app/style/variables.js]: Let's put this in a file in `components/` and then use it in place of the various `Text` components created across the app.

Comment (LanesGood) [reply app/style/variables.js]: I've added the `react-native-global-props` package to set the style across the whole app

**Comment by sethvincent** [comment]
> Implement working icons in FeatureDetailHeader for the View and Add point screens
Taking care of this in separate PRs

**Comment by sethvincent** [comment]
Debugging on ios now and there are some small layout issues around `Add a field` and `Additional tags` that I'll fix up and then it'll be good to merge.

**Comment by LanesGood** [comment]
@sethvincent @kamicut I've set this PR as ready to review, as its gotten a bit large and away from the original stated purpose. I'll work on validation separately, and some of the rest of the app styling separately as well.

This does still need to be tested and debugged on IOS.

**Comment by sethvincent** [comment]
tested and made some improvements for ios/iphonex layout of the tag editor and `add a field` chooser.

**Pull request merged**